Code/CMake/Externals/TINYXML2.cmake

```diff
@@ -31,12 +31,17 @@
 #-----------------------------------------------------------------------------
 # TINYXML2
 set(proj TINYXML2)
+if(LINUX)
+  set(proj tinyxml2)
```

Comment:
What is the reason for needing the lower case on linux?

Reply:
Output below on Ubuntu 14.  Basically, config file is tinyxml2Config.cmake, which doesn't resolve due to case sensitivity.

-- PKG HDF5 found!
CMake Error at CMake/SimVascularMacros.cmake:142 (find_package):
  By not providing "FindTINYXML2.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "TINYXML2",
  but CMake did not find one.

  Could not find a package configuration file provided by "TINYXML2" with any
  of the following names:

    TINYXML2Config.cmake
    tinyxml2-config.cmake

  Add the installation prefix of "TINYXML2" to CMAKE_PREFIX_PATH or set
  "TINYXML2_DIR" to a directory containing one of the above files.  If
  "TINYXML2" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMake/Externals/TINYXML2.cmake:56 (simvascular_external)
  CMake/SimVascularExternals.cmake:37 (include)
  CMakeLists.txt:113 (include)


-- Configuring incomplete, errors occurred!


while the contents of lib/cmake/tinyxml2 is:

-rw-r--r-- 1 simvascular simvascular   57 Jun 20 13:19 tinyxml2Config.cmake
-rw-r--r-- 1 simvascular simvascular 3046 Jun 20 13:19 tinyxml2Targets.cmake
-rw-r--r-- 1 simvascular simvascular  814 Jun 20 13:19 tinyxml2Targets-release.cmake

Reply:
Okay, makes sense